cocos/gfx/base/device.ts

```diff
@@ -335,6 +336,15 @@ export abstract class Device {
      */
     public abstract copyTexImagesToTexture (texImages: Readonly<TexImageSource[]>, texture: Texture, regions: Readonly<BufferTextureCopy[]>): void;
 
+    /**
+     * @en Copy image assets to texture.
+     * @zh 拷贝图像资产到纹理。
+     * @param imageAssets The image assets to be copied.
+     * @param texture The texture to copy to.
+     * @param regions The region descriptions.
+     */
+    public abstract copyImagesToTexture (imageAssets: Readonly<ImageAsset[]>, texture: Texture, regions: Readonly<BufferTextureCopy[]>): void;
```

Comment:
@bluesky013 please take a look.

Reply:
`ImageAsset` is a concept in the rendering layer, and gfx is below the rendering layer. It should not depend on `ImageAsset`. It may be necessary to encapsulate the data structure of gfx.